clusterloader2/testing/load/config.yaml

```diff
@@ -17,12 +17,12 @@
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
 {{$PROMETHEUS_SCRAPE_KUBE_PROXY := DefaultParam .PROMETHEUS_SCRAPE_KUBE_PROXY true}}
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
-{{$ENABLE_CONFIGMAPS := DefaultParam .CL2_ENABLE_CONFIGMAPS false}}
-{{$ENABLE_DAEMONSETS := DefaultParam .CL2_ENABLE_DAEMONSETS false}}
-{{$ENABLE_JOBS := DefaultParam .CL2_ENABLE_JOBS false}}
+{{$ENABLE_CONFIGMAPS := DefaultParam .CL2_ENABLE_CONFIGMAPS true}}
+{{$ENABLE_DAEMONSETS := DefaultParam .CL2_ENABLE_DAEMONSETS true}}
+{{$ENABLE_JOBS := DefaultParam .CL2_ENABLE_JOBS true}}
 {{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS false}}
```

Comment:
Let' disable it back and send another PR in test-infra to change the kubemark presubmit jobs, we forgot about them. It's this file - https://github.com/kubernetes/test-infra/blob/6cb0718bf7f94883199ea290284155b29523db98/config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml#L417